docs/source/traits_user_manual/internals.rst

```diff
@@ -185,7 +185,8 @@ lists, dictionaries and sets.  To achieve this Traits provides special
 subclasses of the standard Python classes that can validate elements and can
 fire trait notifications when the contents change.  These classes are,
 respectively, |TraitList|, |TraitDict| and |TraitSet| (not to be confused with
-the deprecated Trait Handlers of the same names).
+``TraitHandler`` subclasses of the same names, which were removed in
```

Comment:
This isn't quite accurate, since there was never a `TraitSet` subclass of `TraitHandler`. It's probably clearer just to remove this whole parenthetical.